lib/schemapack.js

```diff
@@ -235,7 +239,7 @@ class SchemaPack {
 
   // convert method, this is called when you want to convert a schema that you've already loaded
   // in the constructor
-  convert (callback) {
+  async convert (callback) {
```

Comment:
I'm not sure why are we changing the nature of APIs (we are making it `async`), this wasn't mentioned in the document AFAIK.